freud/diffraction.py

```diff
@@ -691,6 +691,12 @@ def compute(self, system, view_orientation=None, zoom=4, peak_width=1, reset=Tru
             peak_width (float, optional):
                 Width of Gaussian convolved with points, in system length units
                 (Default value = 1).
+            weights ((:math:`N`,) :class:`numpy.ndarray`, optional):
+                SLD weights for each point in the system. If provided, the
+                diffraction pattern will be normalized such that the sum of the
+                weights is equal to the number of points in the system, resulting in
+                :math:`S(0) = N`. If not provided, all points are assumed to
+                have a weight of 1. (Default value = :code:`None`).
```

Comment:
This is the first mention of SLD. Give the full meaning of the acronym when you first mention it.